### ISSUE TITLE
Does apparently not work with Windows paths

### ISSUE THREAD

**Issue by jiai** [hupfdule/nb-edit-externally#1]
I want to use it with Notepad++, but it seems like the plugin can't handle Windows paths correctly. Probably a lot of stuff has to be quoted. At the latest the file path with the file to be edited contains backslashes, often special characters and spaces are also included. I have tried the following:

"C:\Program Files\Notepad++\notepad++.exe" -multiInst -n${line} -c${column} "${file}"

Program path with backslashes does not work, slashes work.
${file} with and without quotes does not work.

**Comment by hupfdule** [comment]
Thank's for your report.

I will look into it to see what I can do about it.

**Comment by hupfdule** [comment]
I found the problem. The way the command line was parsed stripped the backslashes from the path of the file to edit.
I have uploaded version 1.0.1 that fixes that issue.

The backslashes in the path to notepad++.exe still must be escaped by additional backslashes or forward slashes can be uses as you already found out. Therefore both variants will work:

````
"C:\\Program Files\\Notepad++\\notepad++.exe" -multiInst -n${line} -c${column} "${file}"
````
````
"C:/Program Files/Notepad++/notepad++.exe" -multiInst -n${line} -c${column} "${file}"
````

I have updated the documentation to more clearly express this fact when entering paths for MS Windows.
